### PR TITLE
Project parameter for Cloudformation template

### PR DESCRIPTION
**Pull request by pocke**
Goal
-------

- [x] Use a project parameter in Cloudformation template.
- [x] refactor
- [x] Completion perameters

### CODE REVIEW

File: app/models/project_parameter.rb [hunk 1]
```diff
@@ -10,4 +10,21 @@ class ProjectParameter < ActiveRecord::Base
...
+    # @param [String] target
+    # @param [Project] project
+    # @param [Integer] project_id
+    # @return [String]
+    def exec(target, project: nil, project_id: nil)
+      raise ArgumentError if !(project_id || project)
+
+      params = self.where(project_id: project_id || project.id)
+      re = /(?<!\\)\$\{(?<key>#{params.map(&:key).join('|')})\}/
+      target.gsub(re) do |matched|
```
Comment (skyarch-networks) [inline app/models/project_parameter.rb]: __[rubocop]__
* Lint/UnusedBlockArgument: Unused block argument - `matched`. You can omit the argument if you don't care about it. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)

**Comment by pocke** [comment]
@Joeper214 please review && merge it :bow:

File: app/models/project_parameter.rb (hunk 1 above)
Comment (pocke) [inline app/models/project_parameter.rb]: I'll fix it

**Comment by Joeper214** [comment]
I'll review now.

**Comment by Joeper214** [comment]
LGTM!

**Pull request merged**